src/client/src/routes/SimpleLauncher/Link.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import { styled, Styled } from 'rt-theme'
+
+declare const global: any
+
+export interface LinkProps {
+  is?: React.ComponentType<any>
```

Comment:
What do you think of calling this `as`?

Reply:
You can also do something like `React.ReactNode` or `JSX.Element`

Reply:
`is` already exists as a convention for redefining the final component used in the render.

On a note of implied ontological mean `as` suggests a `role` change and to me would mean a less literal application of the supplied value.

The correct change to the typing would actually be to mark it as `string | React.ComponentType<any>` — commonly it is used for `is="a"` or `is="button"`.